hzi-braunschweig/SORMAS-Project
# prompt for unsaved address data (#4477)

**For review / testing:**
Please make also sure that the bugfix from #3014 (fixed in PR #4097) is still working. The bug described in #3014 manifested under these conditions:
- `country.locale=en-US` in `sormas.properties`
- browser language (not SORMAS user language!) set to German
- in a case or its person, fill one of the latitude / longitude fields.
- reload or navigate away and return to this page. Decimal fractions should be displayed with a `,`, e.g., `3,2` now
- navigate away from the page without any prior modification. No prompt to save or discard should pop up.

Additional cleanup:
Commented code removed. The code was commented in 3/26/19, so I would not expect it to be a candidate to be revived.
